device_e2e/aio/stress_results.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+import threading
+import math
+
+
+class Measurements(object):
+    """Object which defines the measurements we take durring a stress test"""
+
+    def __init__(self):
+        self.lock = threading.Lock()
+        self.peak_reconnect_time = 0
+        self.peak_resident_memory_mb = 0
+        self.peak_telemetry_arrival_time = 0
+        self.queued_telemetry_messages = 0
+        self.peak_queued_telemetry_messages = 0
```

Comment:
`queued_telemetry_messages` might sound like the number of messages that were _ever_ in the queue rather than the number that is _currently_ in the queue
```suggestion
        self.messages_in_queue= 0
        self.messages_in_queue = 0
```